src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitTextFieldTests.cs

```diff
@@ -0,0 +1,267 @@
+            var visualClass = visual == Visual.Cupertino ? "cupertino" : visual == Visual.Material ? "material" : "fluent";
+
+            Assert.IsTrue(bitTextField.ClassList.Contains($"bit-txt-{isEnabledClass}-{visualClass}"));
+
+            Assert.AreEqual(isMultiLine ? "TEXTAREA" : "INPUT", bitTextField.FirstElementChild.TagName);
+        }
+
+        [DataTestMethod,
+           DataRow(true, "hello world"),
+           DataRow(false, "hello world")
+       ]
+        public void BitTextFieldShouldTaketValue(bool isMultiLine, string value)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.Value, value);
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                });
+
+            var bitTextField = component.Find(".bit-txt");
+
+            if (isMultiLine)
+            {
+                Assert.AreEqual(bitTextField.FirstElementChild.InnerHtml, value);
+            }
+            else
+            {
+                Assert.AreEqual(bitTextField.FirstElementChild.GetAttribute("value"), value);
+            }
+        }
+
+        [DataTestMethod,
+            DataRow(15, true),
+            DataRow(15, false)
+        ]
+        public void BitTextFieldShouldTakeMaxLength(int maxLength, bool isMultiLine)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.MaxLength, maxLength);
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                });
+
+            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            Assert.IsTrue(bitTextField.HasAttribute("maxlength"));
+            Assert.AreEqual(bitTextField.GetAttribute("maxlength"), maxLength.ToString());
+        }
+
+        [DataTestMethod,
+            DataRow("this is placeholder", true),
+            DataRow("this is placeholder", false),
+        ]
+        public void BitTextFieldShouldTakePlaceHolder(string placeholder, bool isMultiLine)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.Placeholder, placeholder);
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                });
+
+            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            Assert.IsTrue(bitTextField.HasAttribute("placeholder"));
+
+            Assert.AreEqual(bitTextField.GetAttribute("placeholder"), placeholder);
+        }
+
+        [DataTestMethod,
+            DataRow(true),
+            DataRow(false),
+        ]
+        public void BitTextFieldShouldRespectReadOnly(bool isMultiLine)
```

Comment:
please merge this method with the previous test methods

Reply:
done